recipes-kernel/linux/files/bcm2712-raspberrypi5-xen.dtso

```diff
@@ -59,3 +59,75 @@
 &bcm_reset      { xen,passthrough; };
 &pcie_rescal    { xen,passthrough; };
 &mip0           { xen,passthrough; };
+
+&mip0 {
+			interrupts =
```

Comment:
Could you please include 
```
#include <dt-bindings/interrupt-controller/arm-gic.h>
```

And the list interrupts in form `<GIC_SPI 128 IRQ_TYPE_EDGE_RISING>`?

Reply:
It can't be build manually in this case :( (which I use frequently) Therefore all overlays are raw. I think it has to be done as separate change  - if really required.